web/app/integration/wdio-sauce.conf.js

```diff
@@ -2,10 +2,19 @@ exports.config = {
   runner: 'local',
   user: process.env.SAUCE_USERNAME,
   key: process.env.SAUCE_ACCESS_KEY,
+  service: ['sauce'],
   sauceConnect: true,
   specs: [
-      './integration/specs/*.js'
+      './integration/specs/**/*.js'
   ],
+    suites: {
+    sidebar: [
+        './integration/specs/sidebar/*.js'
+    ],
+    daemonset: [
+        './integration/specs/daemonset/*.js'
```

Comment:
According to our [.eslintrc](https://github.com/linkerd/linkerd2/blob/master/web/app/.eslintrc), This line, and a couple of other lines in this file, need to be indented with two spaces instead of four
```diff
diff --git a/web/app/integration/wdio-sauce.conf.js b/web/app/integration/wdio-sauce.conf.js
index ca23c4c6..211ea9cd 100644
--- a/web/app/integration/wdio-sauce.conf.js
+++ b/web/app/integration/wdio-sauce.conf.js
@@ -5,24 +5,24 @@ exports.config = {
   service: ['sauce'],
   sauceConnect: true,
   specs: [
-      './integration/specs/**/*.js'
+    './integration/specs/**/*.js'
   ],
-    suites: {
+  suites: {
     sidebar: [
-        './integration/specs/sidebar/*.js'
+      './integration/specs/sidebar/*.js'
     ],
     daemonset: [
-        './integration/specs/daemonset/*.js'
+      './integration/specs/daemonset/*.js'
     ]
-},
+  },
   // Patterns to exclude.
   exclude: [
-      // 'path/to/excluded/files'
+    // 'path/to/excluded/files'
   ],
   maxInstances: 10,
   capabilities: [
-    {browserName: 'firefox', platform: 'Windows 10', version: '60.0'},
-    {browserName: 'chrome', platform: 'OS X 10.13', version: '69.0'}
+    { browserName: 'firefox', platform: 'Windows 10', version: '60.0' },
+    { browserName: 'chrome', platform: 'OS X 10.13', version: '69.0' }
   ],
   bail: 0,
   baseUrl: 'http://localhost',
@@ -31,7 +31,7 @@ exports.config = {
   connectionRetryCount: 3,
   framework: 'mocha',
   mochaOpts: {
-      ui: 'bdd',
-      timeout: 6000000
+    ui: 'bdd',
+    timeout: 6000000
   }
 }

```

Reply:
Oh this is strange. I don't remember changing anything under capabilities, but yes I'll make the appropriate indents.